### PR TITLE
configure tox

### PR DESCRIPTION
**Pull request by graingert**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,93 @@
...
+# virtualenv
+venv/
+ENV/
+
+# Spyder project settings
+.spyderproject
+
+# Rope project settings
+.ropeproject
+
```
Comment (peterbe) [inline .gitignore]: All of these are not specific to `hashin`. This looks like your personal preferences. Then you should put them in `~/.gitignore_global` or something. 
See https://www.peterbe.com/plog/gitinfoexclude :)

Comment (graingert) [inline .gitignore]: Created by https://www.gitignore.io/api/python

**Comment by peterbe** [comment]
Awesome! If you can just undo your changes to the `.gitignore` we'll land this.

**Comment by graingert** [comment]
@peterbe This is the default .gitignore from https://www.gitignore.io/api/python

**Comment by peterbe** [comment]
Interesting, but I prefer to not have that in my project's `.gitignore`. But the list it neat! I might copy some of that to my `.gitignore_global`.

**Comment by peterbe** [comment]
I think it's better if I just fix it myself. You've helped so much already.

**Pull request merged**